api/nodes/views.py

```diff
@@ -11,6 +11,11 @@
 from .serializers import NodeSerializer, NodePointersSerializer, NodeFilesSerializer
 from api.users.serializers import ContributorSerializer
 from .permissions import ContributorOrPublic, ReadOnlyIfRegistration, ContributorOrPublicForPointers
+from django.utils.translation import ugettext_lazy as _
```

Comment:
This should be moved to the block containing `from modularodm import Q`, as it's a third-party library.